enhancements/ingress/ingress-dashboard.md

```diff
@@ -0,0 +1,199 @@
+
+## Motivation
+
+While ingress related metrics already exist and are accessible in the OpenShift Console (via the menu “Observe” > Metrics”), there is no consolidated view presenting a summary of them. There are existing dashboards in other areas (such as etcd, compute resources, etc.), but networking today is less represented there, despite its importance for monitoring and troubleshooting.
+
+In addition, product management has shown interest in providing and making more visible some cluster-wide statistics such as the number of routes and shards in use.
+
+This [request for enhancement](https://issues.redhat.com/browse/NE-1059) is also considered here, although it will not be entirely fulfilled as we don't intend here to create any new metric.
+
+### Goals
+
+1. Review and select a subset of the available metrics that should be included in the new dashboard. This should ideally be done with the help of the NetEdge team expertise and/or SREs.
+2. Design a new dashboard with these metrics.
+3. Update the Ingress Operator to deploy this dashboard, making it accessible for Cluster Monitoring consumption.
+
+### Non-Goals
+
+- This work is not intended to provide a comprehensive set of metrics all at once. Instead, the intent is to "start small", setting in place all the mechanisms in code, and iterate later based on feedback to add or amend elements in the dashboard without any code change.
+- This enhancement does not include any new metric creation or exposition: only already available metrics are considered. If discussions lead to consider the creation of new metrics, this could be the purpose of a follow-up enhancement.
+
+### User Stories
+
+> As a cluster administrator, I want to get a quick overview of general cluster statistics, such as the number of routes or shards in use.
+> As a cluster administrator, I want to get a quick insight in incoming traffic statistics, such as on latency and HTTP errors.
+
+## Proposal
+
+To make a new dashboard discoverable by Cluster Monitoring, a `ConfigMap` needs to be created in the namespace `openshift-config-managed`, containing a static dashboard definition in Grafana format (JSON). The dashboard datasource has to be Cluster Monitoring's Prometheus.
+
+The Ingress Operator is responsible for creating and reconciling this `ConfigMap`. We assume all metrics used in the dashboard are present unconditionally, which allows us to create a static dashboard unconditionally as well.
+In the event where conditional display should be introduced later (e.g. an alternative to HAProxy being implemented, resulting is different metrics to display), different approaches can be discussed: the dashboard could be dynamically amended via JSON manipulation; or several dashboards could be embedded and selectively installed.
```

Comment:
I think this should be pointed out as a goal or non-goal based on the commitment.

Reply:
This paragraph is not about what is going to be done here but more an hypothetical suggestion about what could be done if at some point there is a need for dashboard with dynamic content.

We ca remove the paragraph if you prefer.